### DISCUSSION TITLE
Windows 11 KB5008353 - Explorer totally won't load

### DISCUSSION THREAD

**Discussion by AMDMan64** [valinet/ExplorerPatcher#691] [General]
With the latest build 22000.466 - Explorer totally won't load while ExplorerPatcher is installed.   After rebooting for the update, I ended up with just the desktop background - nothing else.  Explorer was running, but not working.  I had to run ep_setup.exe /uninstall to make the system work again.  This build is apparently Beta / Release Preview channel currently, but likely will be rolling out to general release soon.
Here is a listing of all the changes:
https://blogs.windows.com/windows-insider/2022/01/14/releasing-windows-11-build-22000-466-to-beta-and-release-preview-channels/
Hopefully ExplorerPatcher can be fixed to work with this new build.  For now, I'm going to remove the pre-release update because I can't stand using my system without full taskbar labels.

**Comment by kichrot** [comment]
I have 22000.466 ExplorerPatcher working normally.

**Reply by AMDMan64** [reply]
It apparently works with the prerelease build, but not with the one that I had.  I reloaded it tonight with 222000.469.42.0 of ExplorerPatcher and it is loading up again now.
